src/dashboard/src/components/filesystem_ajax/views.py

```diff
@@ -622,20 +624,37 @@ def create_arrange_directory(path):
     )
 
 
+def create_arrange_directories(paths):
+    arranges = [
+        models.SIPArrange(
+            original_path=None,
+            arrange_path=os.path.join(path, ""),  # ensure ends with /
+            file_uuid=None,
+        )
+        for path in paths
+        if path.startswith(DEFAULT_ARRANGE_PATH)
+    ]
+    models.SIPArrange.objects.bulk_create(arranges, BULK_CREATE_BATCH_SIZE)
+
+
 def create_directory_within_arrange(request):
     """ Creates a directory entry in the SIPArrange table.
 
     path: GET parameter, path to directory in DEFAULT_ARRANGE_PATH to create
     """
     error = None
 
-    path = base64.b64decode(request.POST.get("path", ""))
-
-    if path:
-        try:
-            create_arrange_directory(path)
-        except ValueError as e:
-            error = str(e)
+    if "path[]" in request.POST:
```

Comment:
I'm not sure why the behaviour needs to be different with one vs many directories. It seems like we could just always do `request.POST.getlist("path", []))`, and we don't need both `create_arrange_directories` and `create_arrange_directory`

Reply:
It doesn't. I removed `create_arrange_directory` and updated the callers.